src/stactools/chesapeake/utils.py

```diff
@@ -70,3 +73,33 @@ def create_tiles(left: float,
         x += size
         y = bottom
     return tiles
+
+
+def remove_nodata(indir: str, nodata_dir: str) -> None:
+    """Removes TIF files that contain only nodata values to a new directory
+    named "nodata_tifs".
+
+    Args:
+        indir (str): Directory containing the TIF files.
+        nodata_dir (Optional[str]): Directory to move TIF files that contain
+            only nodata values.
```

Comment:
This isn't optional in the function signature.

Reply:
Yeah, [I moved the option to the command](https://github.com/pjhartzell/chesapeake/pull/1/files#diff-45149b47b4521c7333a2ebe60f2087d35845eb27677eba5f6fc7aaf3dec26e7dR66), which also creates the default folder if the option is not passed. Failed to communicate that. Any downsides to this?

Reply:
I think that's fine -- docs just need an update.
```suggestion
        nodata_dir (str): Directory to move TIF files that contain only nodata values.
```

Reply:
Sounds good. Docs updated in several places.